src/main/java/org/apache/lucene/analysis/kr/KoreanTokenizer.java

```diff
@@ -162,6 +115,13 @@ public final void end() {
 
 	@Override
 	public void reset() throws IOException {
+        super.reset();
+
+		if (scanner == null)
+		{
+			scanner = new KoreanTokenizerImpl(input);
+		}
+
```

Comment:
You're following TripAdvisor style conventions but breaking consistency with the file